pagopa/io-app
# feat(IT Wallet): [SIW-979] Add issuing authentication mode selection screen

## Short description
This PR adds the authentication method selection screen, required for credentials issuing for the IT Wallet.

## List of changes proposed in this pull request
- Updated `@pagopa/io-app-design-system` to **1.36.6**
- Added `ItwIdentificationModeSelectionScreen` component
- Added routes and params

## How to test
Navigate to **Profile > IT Wallet > Issuing**, check that the new screen is correctly rendered.

## Preview

<img src="https://github.com/pagopa/io-app/assets/6160324/a093beed-8bd9-46fd-95d3-868ad93ca130" width="200" />
